libavfilter/vf_libvmaf.c

```diff
@@ -822,3 +1075,19 @@ const AVFilter ff_vf_libvmaf_cuda = {
     .flags_internal = FF_FILTER_FLAG_HWFRAME_AWARE,
 };
 #endif
+
+#if CONFIG_LIBVMAF_METADATA_FILTER
```

Comment:
This is unneeded. Not sure if this was intentional, or just a copy/paste error.

Reply:
Yes this is copy/paste error.